cni/pkg/iptables/iptables.go

```diff
@@ -455,7 +462,7 @@ func (cfg *IptablesConfigurator) CreateHostRulesForHealthChecks(hostSNATIP, host
 
 	log.Info("Adding host netnamespace iptables rules")
 
-	if err := cfg.executeCommands(builder); err != nil {
+	if err := cfg.executeCommands(log.WithLabels("component", "host"), builder); err != nil {
```

Comment:
:+1: 

Should we add `log.WithLabels("component", "pod")` or similar for the other flow in the pod namespace?

Reply:
they already have the pod info so the can be distinguished. Not sure it adds a ton? maybe if someone is going to be querying these logs but that seems unlikely for these types of logs